ColabTool/Shared/Post/index.jsx

```diff
@@ -3,8 +3,8 @@ import {
   Card,
 } from '@bufferapp/components';
 import style from './style.css';
-import PostButtonPanel from '../PostButtonPanel';
 import PostDetails from '../PostDetails';
+import PostTopDetails from '../PostTopDetails';
```

Comment:
Ah, good one, @hharnisc - those names feel good to me 😄  Naming things is the trickiest part 😂 Appreciate your thoughts here! Thank you!